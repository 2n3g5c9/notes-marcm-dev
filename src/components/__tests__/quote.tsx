import React from 'react'
import renderer from 'react-test-renderer'

import { Quote } from '@components'

jest.mock('gatsby-plugin-mdx', () => {
  return {
    // eslint-disable-next-line react/display-name
    MDXRenderer: ({ children }: { children: React.ReactNode }) => {
      return <div>{children}</div>
    },
  }
})

describe('Quote', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Quote>Test</Quote>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
