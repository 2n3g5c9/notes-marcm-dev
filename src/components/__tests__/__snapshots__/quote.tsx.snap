// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders correctly 1`] = `
<div
  className="flex mt-6 sm:mt-7"
>
  <div
    className="flex-shrink-0 mb-4 self-center"
  >
    <test-file-stub
      className="h-32 sm:h-36 mr-10 lg:mr-0"
    />
  </div>
  <div
    className="flex md:flex-grow justify-center self-center"
  >
    <div>
      <test-file-stub
        className="absolute h-9 sm:h-10 opacity-20 transform -translate-x-6 -translate-y-1 sm:-translate-y-2 w-9 sm:w-10"
      />
    </div>
  </div>
</div>
`;
